programs/eosio-blocklog/main.cpp

```diff
@@ -233,6 +237,12 @@ bool trim_blocklog_front(bfs::path block_dir, uint32_t n) {        //n is first
    return status;
 }
 
+void fix_irreversible_blocks(bfs::path block_dir) {
+    std::cout << "\nfix_irreversible_blocks of blocks.log and blocks.index in directory " << block_dir << '\n';
+    block_log block_logger(block_dir, true);
```

Comment:
Just use block_log block_logger(block_dir, fc::path(), 0, 0, true);

Reply:
removed it.